py3status/modules/spotify.py

```diff
@@ -150,18 +151,22 @@ def _get_text(self):
                 )
                 if playback_status.strip() == "Playing":
                     color = self.py3.COLOR_PLAYING or self.py3.COLOR_GOOD
+                    state = "playing"
                 else:
                     color = self.py3.COLOR_PAUSED or self.py3.COLOR_DEGRADED
+                    state = "paused"
             except Exception:
                 return (
-                    self.format_stopped,
+                    self.py3.safe_format(self.format_stopped, dict(state="stopped")),
```

Comment:
(Opinion) I don't think we would need to supply an artificial `Stopped` string because of implied `format_stopped`. If everything are passed to a single `format`, then yeah, maybe sure.

Reply:
You are absolutely right. If it was for me I would ditch the `format_stopped` and use single `format`, I think it would match your simplistic approach.
But of course I need your opinion on this, I don't even now if we can do changes that aren't backwards compatible.

Reply:
Ditching `format_stopped` would be nice. :-)

For backward compatibility, don't give a dict. Users can add a word `Stopped` or a stop symbol to their `format_stopped`  instead.  I assume `{playback}` will be reserved for `format` only... and it gives no more than `Playing` or `Paused`.